packages/collector/test/tracing/cloud/aws-sdk/v3/kinesis/test_definition.js

```diff
@@ -0,0 +1,239 @@
+          });
+        } else {
+          describe(`getting result with error: ${withError ? 'yes' : 'no'}`, () => {
+            availableOperations.forEach(operation => {
+              const requestMethod = getNextCallMethod();
+              it(`operation: ${operation}/${requestMethod}`, async () => {
+                const withErrorOption = withError ? '?withError=1' : '';
+                const apiPath = `/${operation}/${requestMethod}`;
+                const response = await appControls.sendRequest({
+                  method: 'GET',
+                  path: `${apiPath}${withErrorOption}`,
+                  simple: withError === false
+                });
+                if (operation === 'createStream') {
+                  await checkStreamExistence(streamName, true);
+                }
+                return verify(appControls, response, apiPath, operation, withError);
+              });
+            });
+          });
+        }
+      });
+
+      function verify(controls, response, apiPath, operation, withError) {
+        return retry(
+          () => agentControls.getSpans().then(spans => verifySpans(controls, spans, apiPath, operation, withError)),
+          retryTime
+        );
+      }
+      function verifySpans(controls, spans, apiPath, operation, withError) {
+        const httpEntry = verifyHttpRootEntry({ spans, apiPath, pid: String(controls.getPid()) });
+        verifyExitSpan({
+          spanName: 'kinesis',
+          spans,
+          parent: httpEntry,
+          withError,
+          pid: String(controls.getPid()),
+          extraTests: [
+            span =>
+              expect(span.data.kinesis.op).to.equal(
+                withError && operationsInfo[operation].match(/^getRecords$|^shardIterator$/)
+                  ? 'listShards'
+                  : operationsInfo[operation]
+              ),
+            span =>
+              expect(span.data.kinesis.stream).to.equal(
+                !span.data.kinesis.op.match(/^getRecords$|^listStreams$/) ? streamName : undefined
+              ),
+            span => {
+              if (span.data.kinesis.op === 'shardIterator') {
+                expect(span.data.kinesis.shard).to.equal('shardId-000000000000');
+                expect(span.data.kinesis.shardType).to.equal('AT_SEQUENCE_NUMBER');
+                expect(span.data.kinesis.startSequenceNumber).to.exist;
+              }
+            }
+          ]
+        });
+
+        if (!withError) {
+          verifyHttpExit({ spans, parent: httpEntry, pid: String(controls.getPid()) });
+        }
+      }
+    });
+
+    describe('tracing disabled', () => {
+      this.timeout(config.getTestTimeout() * 2);
+
+      const appControls = new ProcessControls({
+        appPath: path.join(__dirname, 'app'),
+        useGlobalAgent: true,
+        tracingEnabled: false,
+        env: {
+          AWS_KINESIS_STREAM_NAME: streamName,
+          AWS_SDK_CLIENT_REQUIRE: version
```

Comment:
question: Is `AWS_SDK_CLIENT_REQUIRE` used? 🤔

Reply:
Removed